### ISSUE TITLE
New repository for freetype-py ?

### ISSUE THREAD

**Issue by rougier** [rougier/freetype-py#111]
Maybe we could have a dedicated organization (freetype ?) where this repo could be moved. 
I'm ok keeping the repo under my repo but since I'm not the most active anymore, it might be a better solution in the long term.

What do you think ?

**Comment by HinTak** [comment]
There hasn't been any big disagreement on how/what freetype-py works between the people who are contributing, so there is no strong need for any organizational structure yet. Though some of the others are probably funded by some org to work on freetype-py, and may want more visibility/acknowledgement. (I am mildly against it moving under a big corporation's umbrella).

**Comment by rougier** [comment]
By organization, I mean only a GitHub organization. But if everyone's fine with the current situation, I'm fine too.

**Comment by madig** [comment]
There's the FreeType upstream GitHub organization: https://github.com/freetype. Werner is still wary of newfangled development platforms, so the offical repo is not on GitHub. I guess placing it there would give freetype-py an official seal of approval, which would be ok I guess? @cherusker

**Comment by cherusker** [comment]
Right, https://github.com/freetype exists and is "officially maintained by FreeType". Moving `freetype-py` over there seems like a good idea to me but @lemzwerg (Werner) has to agree to that first. Since I'm not sure how active Werner is on GitHub, @rougier could you maybe shoot a message to the devel mailing list (https://lists.nongnu.org/mailman/listinfo/freetype-devel) and kindly ask for Werner's approval? That way everyone else in that list can also voice their opinion but I don't think there will be much (if any) resistance.

@HinTak I hear you but can't totally follow you. Could you please elaborate on your point why it's a bad idea to make `freetype-py` a little bit more official? As far as I am concerned, pure-private repos are used for prototyping while org repos are used for spreading software that is (or was at some point) legitimately ready for distribution due to somewhat higher credibility.

**Comment by lemzwerg** [comment]
If you want to move this is OK with me.  IMHO, it doesn't really matter where the repository is located, so it is fully OK with me if you don't move :-)

That I don't migrate FreeType to github has more philosophical reasons than practical ones...

**Comment by HinTak** [comment]
I am okay with moving to https://github.com/freetype. I am against moving to google or googlefonts (which some of the other contributors are affiliated to).

**Comment by cherusker** [comment]
I see and totally agree, @HinTak.

I have just granted both of you, @rougier and @HinTak admin rights to https://github.com/freetype; feel free to move `freetype-py` whenever you feel ready 👍

**Comment by rougier** [comment]
@cherusker Thanks. Can you also add @madig ?

**Comment by cherusker** [comment]
@madig is already part of that org 😄

**Comment by Korijn** [comment]
@rougier maybe it's time to make the move?

I would also really like to help maintain, together with @almarklein as our projects are critically dependent on this library.

**Comment by rougier** [comment]
Well actually, I don't think there are new reason to move.
